Challenges/Arrays/ArrayShift.cs

```diff
@@ -4,8 +4,19 @@ namespace Challenges.Arrays
 {
     public class ArrayShift
     {
-        public ArrayShift()
+        public ArrayShift
         {
+          (ArrayShift [0] = 2;
+          ArrayShift [1] = 4;
+          ArrayShift [2] = 6;
+          ArrayShift [3] = 8;
+          ArrayShift [4] = 5;),
+
+          (ArrayShift[0] = 4;
+          ArrayShift[1] = 8;
+          ArrayShift[2] = 23;
+          ArrayShift[3] = 42;
+          ArrayShift[4] = 15;)
```

Comment:
All of this is not what you should be trying to do inside of this _method_. This is where you will right the actual logic to insert the given value into the given array.